src/action/base/setup_default_profile.rs

```diff
@@ -1,12 +1,10 @@
 use std::path::PathBuf;
 
 use crate::{
-    action::{ActionError, ActionErrorKind, ActionTag, StatefulAction},
+    action::{common::ConfigureNix, ActionError, ActionErrorKind, ActionTag, StatefulAction},
```

Comment:
I wonder about this from organizational perspective, because it surprised me a little to see something from a "common" action used in a "base" action?

A quick grep didn't turn up something that made me 100% certain about my expectation here. It sounds a little like the first line of src/action/common/mod.rs, `//! [`Action`](crate::action::Action)s which only call other base plugins`, may be talking about this--but the "other base plugins" part throws me a little, and the comment at the top of src/action/mod.rs didn't help talk me down.

Aside: is this bit where you're shuffling some code between the base and common action the refactoring you mentioned in the meeting? I'll lay grappling with the diff of these two files for now.

Reply:
Reviewing it this should probably be a common action ^^

Reply:
This one should probably be in common, but since it wasn't added in this PR, it should probably just be left here for now